src/Illuminate/Foundation/Testing/Concerns/InteractsWithDatabase.php

```diff
@@ -13,15 +13,15 @@ trait InteractsWithDatabase
     /**
      * Assert that a given where condition exists in the database.
      *
-     * @param  string  $table
+     * @param  string|Model $table
```

Comment:
Invalid phpdoc.

Reply:
Please only use the fully qualified names. In Laravel, the phpdoc is there to help people, not just machines.

Reply:
Depending on the IDE and its settings all the people have to do is hovering "Model" with the mouse pointer to see the full qualified name. :P

Reply:
Updated.